seqr_scripts/seqr_sample_qc.py

```diff
@@ -0,0 +1,530 @@
+        type=int,
+        required=True,
+    )
+    parser.add_argument(
+        "--data-source",
+        help="Data source (Internal or External)",
+        choices=["Internal", "External"],
+        required=True,
+    )
+    parser.add_argument(
+        "--is-test",
+        help="To run a test of the pipeline using test files and directories",
+        action="store_true",
+    )
+    parser.add_argument("--sharded", help="Callset VCF is sharded", action="store_true")
+    parser.add_argument(
+        "--callrate-low-threshold",
+        help="Lower threshold at which to flag samples for low callrate",
+        default=0.85,
+    )
+    parser.add_argument(
+        "--contam-up-threshold",
+        help="Upper threshold at which to flag samples for elevated contamination",
+        default=5,
+    )
+    parser.add_argument(
+        "--chimera-up-threshold",
+        help="Upper threshold at which to flag samples for elevated chimera",
+        default=5,
+    )
+    parser.add_argument(
+        "--wes-coverage-low-threshold",
+        help="Lower threshold at which to flag exome samples for low coverage",
+        default=85,
+    )
+    parser.add_argument(
+        "--wgs-coverage-low-threshold",
+        help="Lower threshold at which to flag genome samples for low coverage",
+        default=30,
+    )
+    parser.add_argument(
+        "--plat-min-cluster-size",
+        help="Minimum cluster size for platform pca labeling",
+        default=40,
+    )
+    parser.add_argument(
+        "--plat-min-sample-size",
+        help="Minimum sample size for platform pca labeling",
+        default=40,
+    )
+    parser.add_argument(
+        "--pop-assignment-pcs",
+        help="Number of principal components to use in population assignment RF",
+        default=6,
+    )
+    parser.add_argument(
+        "--plat-assignment-pcs",
+        help="Number of principal components to use in platform assignment clustering",
+        default=6,
+    )
+    parser.add_argument(
+        "--pop-rf-classifier",
+        help="fit from a previously trained random forest model",
+        default="gs://seqr-datasets/sample_qc_resources/population_assignment/gnomad_cmg.RF_fit_90.pkl",
+    )
+    parser.add_argument(
+        "--skip-write-mt", help="Skip writing out qc mt", action="store_true"
+    )
+    parser.add_argument(
+        "--skip-validate-mt",
+        help="Skip validating the mt against common coding and noncoding variants",
+        action="store_true",
+    )
+    parser.add_argument(
+        "--project-list", help="List of seqr projects that are in the callset"
+    )
+    parser.add_argument(
+        "--slack-channel", help="Slack channel to post results and notifications to."
+    )
+    parser.add_argument(
+        "--overwrite", help="Overwrite previous paths", action="store_true"
+    )
+
+    args = parser.parse_args()
+
+    if args.slack_channel:
+        from slack_creds import slack_token
```

Comment:
basic question, are you assuming you have a `slack_creds.py` locally in the same directory to run this?

Reply:
Sorry I should have a comment for this which I will add in the documentation PR. Yes. I have the path to slack_creds.py in my HAIL_SCRIPTS global in my bash profile to achieve this.